goal_src/engine/ps2/memcard-h.gc

```diff
@@ -32,6 +32,25 @@
   :flag-assert         #x900000044
   )
 
+(defmethod inspect mc-file-info ((obj mc-file-info))
+  (format #t "[~8x] ~A~%" obj 'mc-file-info)
```

Comment:
The compiler autogenerates these so there's no need to add them again, even though they show up in the decompilation.